requirements.txt

```diff
@@ -5,21 +5,21 @@ Babel==2.10.3
 Beaker==1.11.0
 bleach==5.0.1
 blinker==1.5
-boto3==1.28.12
-botocore==1.31.12
+boto3==1.28.16
+botocore==1.31.16
 certifi==2023.7.22
 cffi==1.15.1
 chardet==5.1.0
 charset-normalizer==3.2.0
 ckan==2.10.1
 ckanext-datajson==0.1.20
-ckanext-dcat-usmetadata==0.4.2
+ckanext-dcat-usmetadata==0.5.0
 ckanext-envvars==0.0.3
 ckanext-googleanalyticsbasic==0.2.1
 ckanext-s3filestore @ git+https://github.com/keitaroinc/ckanext-s3filestore.git@caf88c0352ffe7b4432d3d55ddfb0a71249ceddd
 ckanext-saml2auth @ git+https://github.com/GSA/ckanext-saml2auth.git@c2b12a94430034c522b25d282323a064e2d6a03a
 ckanext-usmetadata==0.3.2
--e git+https://github.com/ckan/ckanext-xloader.git@c062f547e8e5dd80fd5dae6d184fe268ccb4b32e#egg=ckanext_xloader
+-e git+https://github.com/ckan/ckanext-xloader.git@16b84175005435d579607638dfe2056dda61af70#egg=ckanext_xloader
```

Comment:
Hmmm...

Reply:
is this something that shouldn't have changed?

Reply:
I'm always concerned about changing requirements... but tests passed, so I don't have reason to complain too much 🤷